modules/network/public-ip-prefixes/README.md

```diff
@@ -166,7 +166,7 @@ The following module usage examples are retrieved from the content of the files
 <summary>via Bicep module</summary>
 
 ```bicep
-module publicIPPrefixes './Network/publicIPPrefixes/main.bicep' = {
+module publicIpPrefixes './network/public-ip-prefixes/main.bicep' = {
```

Comment:
I guess that should remain `publicIPPrefixes`?

Reply:
Good catch! :fishing_pole_and_fish:

I updated the utility for special conversions and regenerated the readmes. This is needed for publicIPPrefixes and publicIPAddresses.